src/Geta.404Handler/lang/RedirectsGadget.xml

```diff
@@ -64,6 +65,7 @@
     <language name="Norsk" id="NO">
         <gadget>
             <redirects>
+                <redirectType>Redirect Type</redirectType>
```

Comment:
Add translated values, it would be enough to translate with Google Translate.

Reply:
Translations added